ironmon_tracker/MoveData.lua

```diff
@@ -1,3756 +1,3768 @@
 		pp = "10",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "Changes weather to Sandstorm for 5 turns. Pokemon take 1/16 of their max HP in damage, except for Steel, Ground, and Rock types.",
 	},
 	{
 		id = "202",
 		name = "Giga Drain",
-		type = PokemonTypes.GRASS,
+		type = PokemonData.Types.GRASS,
 		power = "60",
 		pp = "5",
 		accuracy = "100",
-		category = MoveCategories.SPECIAL,
+		category = MoveData.Categories.SPECIAL,
 		summary = "50% of the damage dealt is restored to the user as HP.",
 	},
 	{
 		id = "203",
 		name = "Endure",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "10",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		priority = "+ 3",
 		summary = "Allows the user to survive any single attack that would cause it to faint, leaving the user with 1 HP instead.",
 	},
 	{
 		id = "204",
 		name = "Charm",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "20",
 		accuracy = "100",
-		category = MoveCategories.STATUS,
+		category = MoveData.Categories.STATUS,
 		summary = "Lowers the Attack stat of the target by two stages.",
 	},
 	{
 		id = "205",
 		name = "Rollout",
-		type = PokemonTypes.ROCK,
+		type = PokemonData.Types.ROCK,
 		power = "30",
 		pp = "20",
 		accuracy = "90",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Deals damage over 5 turns, doubling in power each consectutive hit. Base power is doubled if the user had previously used Defense Curl.",
 	},
 	{
 		id = "206",
 		name = "False Swipe",
-		type = PokemonTypes.NORMAL,
+		type = PokemonData.Types.NORMAL,
 		power = "40",
 		pp = "40",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		iscontact = true,
 		summary = "Deals damage, but will always leave the target with 1 HP if it would otherwise cause it to faint.",
 	},
 	{
 		id = "207",
 		name = "Swagger",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "15",
 		accuracy = "90",
-		category = MoveCategories.STATUS,
+		category = MoveData.Categories.STATUS,
 		summary = "Raises the target's Attack stat by two stages and confuses it.",
 	},
 	{
 		id = "208",
 		name = "Milk Drink",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "10",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "Restores up to 50% of the user's max HP. Can be used outside of battle.",
```

Comment:
"Can be used outside of battle." could be misleading, as the out of battle effect is transferring user's hp to another pokemon not healing the user